### DISCUSSION TITLE
Setting Vercel's Data Caching Location?

### DISCUSSION THREAD

**Discussion by chuchuwiiii** [vercel/community#7013] [Help]
### Summary

Hi. I'm managing a project that uses Singapore (sin1) region as the primary edge region (a member of the team, not the owner). Upon looking at the `data cache read` section under the Usage tab within the Vercel's team page, I found out that most cache are being read and write in Washinton DC (iad1).

I'm wondering if I can change to Singapore? If I can, how can I do it?

<img width="1300" alt="Screenshot 2567-05-28 at 21 52 44" src="https://github.com/vercel/community/assets/57100501/8bbd4396-5274-4b7c-ad21-5fafd46a2612">
<img width="882" alt="Screenshot 2567-05-28 at 21 58 07" src="https://github.com/vercel/community/assets/57100501/b93d70c2-e3f2-430b-9619-550c348bc905">
<img width="891" alt="Screenshot 2567-05-28 at 22 02 41" src="https://github.com/vercel/community/assets/57100501/d04cad6d-3339-4a44-b7fe-9342232131e3">

### Steps to Reproduce

Set function region to Singapore. The data cache is Washinton. Sorry, there's no detailed steps to be written down.

**Comment by antonio-bluco** [comment]
Hi, 

Any updates here? It seems like a pretty common thing to want to do - data cache is enabled by default, and it should really be close to where your users / db are :)

**Reply by pawlean** [reply]
@antonio-bluco Have you set your default region already and can see this same behaviour? See [related documentation](https://vercel.com/docs/functions/configuring-functions/region#project-configuration).

**Reply by Vaishaliydv27** [reply]
Hi @pawlean  

- I have my default function location set to `bom1` as you can see in this screenshot - 
![image](https://github.com/user-attachments/assets/0bc757e4-7ca9-4fec-9e27-a52d52d9f59e)

- I have se this in vercel.json - 
![image](https://github.com/user-attachments/assets/3b6d0978-d6eb-492a-95b9-20bf3b7ee228)


Still, all my data cache reads and writes are from the washington region. How can I fix this?

![image](https://github.com/user-attachments/assets/4ed17cb2-da32-41b9-a826-368dc611273c)

[a comment by an automated account is not shown]
